src/assets/css/header.scss

```diff
@@ -182,7 +182,7 @@
   $largeHeight: 64px;
   $largeWidth: $largeHeight * $aspectRatio;
 
-  background-image: url('/assets/img/logo.svg');
```

Comment:
As a heads-up not starting a path with '/assets/' will break the hashed filename rewriting in the build script unless it's updated to handle relative paths.

Reply:
I suppose this happens in the `asset-pipeline` script that we don't yet have in this repo?